modules/sitemaps/sitemap-stylist.php

```diff
@@ -192,6 +192,12 @@ public static function sitemap_index_xsl() {
 			)
 		);
 
+		if ( current_user_can( 'manage_options' ) ) {
```

Comment:
@jeherve @kraftbj 

This is the new part, ignore all the spaces->tabs :/

Reply:
I haven't tried to figure out why, but spinning up a new JN site with an empty sitemap isn't showing me the `$no_nodes_warning` on the sitemap or in the style.